src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/Content/cart_status_data.html.twig

```diff
@@ -0,0 +1,34 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+<div class="text-{{ column.options.text_align }}">
+  {% if record[column.options.field] matches '/^\\d+$/' %}
```

Comment:
This pattern must be in a variable, unreadable for a lambda dev :trollface:

Reply:
in a twig variable? or this check better moved to php code?

Reply:
I think it can stay in twig as it's a matter of display.
The idea of using a variable is to use the variable name as a description of the "why".

Why is this specific pattern being displayed in a different way than others ?

If for example the reason is "display fields that are an URL like X else like Y" then you would use a twig variable:
```
{% var isFieldLikeAnUrl = record[column.options.field] matches '/^\\d+$/' %}
```

Reply:
Exactly, thanks @matks :)